src/R/Editor/Impl/Signatures/SignatureHelpSource.cs

```diff
@@ -20,9 +21,11 @@
 namespace Microsoft.R.Editor.Signatures {
     sealed class SignatureHelpSource : ISignatureHelpSource {
         private ITextBuffer _textBuffer;
```

Comment:
Done

---
In reply to: [68853191](https://github.com/Microsoft/RTVS/pull/2022#discussion_r68853191) [](ancestors = 68853191)